### ISSUE TITLE
[Question] How can we access data from terraform module inside a terragrunt hcl file ?

### ISSUE THREAD

**Issue by axelczk** [gruntwork-io/terragrunt#2226]
Here is my hcl file:

```
terraform {
  source = "git@...."
}

locals = {
  config  = templatefile("./config.yml", { environment = {NEED HELP HERE } )]])
}

```

Inside the terraform module, I have a data that retrieve a value from our cloud provider. I would like to know if it is possible to use this data in order to file the variable required inside the templatefile ?
The only solution I have is to use an run_cmd in order to reproduce the behavior of the data block in the module.

Thanks for your help,

Have a great day.

**Comment by axelczk** [comment]
This is not what I really want in fact. But I think it's not possible. 

In fact, the hcl file I linked in the issue is used to manage dependency for the module between multiple others modules. This is why I was asking it the parent could use data from the terraform source but I don't think it's possible regarding your response.

I will use an run_cmd like I already did. It was just to find out if there was a better way to do it.
